scipy/special/_evalpoly.h

```diff
@@ -0,0 +1,45 @@
+/* Evaluate polynomials.
+ *
+ * All of the coefficients are stored in reverse order, i.e. if the
+ * polynomial is
+ *
+ *     u_n x^n + u_{n - 1} x^{n - 1} + ... + u_0,
+ *
+ * then coeffs[0] = u_n, coeffs[1] = u_{n - 1}, ..., coeffs[n] = u_0.
+ *
+ * References
+ * ----------
+ * [1] Knuth, "The Art of Computer Programming, Volume II"
+ */
+
+
+#ifndef EVALPOLY_H
```

Comment:
Rather than include guards, we can use `#pragma once` here. See [Pragma once](https://en.wikipedia.org/wiki/Pragma_once). That's more modern, but if someone knows of some weird case it doesn't work please say! I've never experienced any problems with it...